src/classes/CON_PreferredPhone_TEST.cls

```diff
@@ -36,33 +36,32 @@
 */
 @isTest
 private class CON_PreferredPhone_TEST {
-    
-    @testSetup
-    static void dataSetup() {
-        
-        UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Disable_Preferred_Phone_Enforcement__c = False)
+
+    private static Hierarchy_Settings__c enablePreferredPhone() {
```

Comment:
I like that you broke this out into methods as it makes it more clear, especially since we enabling to disable ;) Good idea!